packages/flutter_tools/schema/pubspec_yaml_android.json

```diff
@@ -0,0 +1,214 @@
+          }
+        }
+      },
+      "launcherIcons": {
+        "type": "object",
+        "oneOf": [{
+          "properties": {
+            // densities: https://developer.android.com/training/multiscreen/screendensities
+            "xxxhdpi": { "type": "string" },
+            "xxhdpi": { "type": "string" },
+            "xhdpi": { "type": "string" },
+            "hdpi": { "type": "string" },
+            "mhdpi": { "type": "string" },
+            "lhdpi": { "type": "string" }
+          }
+        }, {
+          "properties": {
+            "svgIcon": { "type": "string" }
+          }
+        }]
+      },
+      // https://developer.android.com/guide/topics/ui/look-and-feel/themes
+      "theme": {
+        "type": "object",
+        "properties": {
+          "colorPrimary": { "type": "string" },
+          "colorPrimaryDark": { "type": "string" },
+          "colorAccent": { "type": "string" }
+        }
+      },
+      "splash": {
+        "type": "object",
+        "oneOf": [{
+          "properties": {
+            "color": { "type": "string" }
+          }
+        },{
+          "properties": {
+            "bitmap": { "type": "string" }
+          }
+        },{
+          "properties": {
+            "drawableResource": { "type": "string" }
+          }
+        }]
+      },
+      // https://developer.android.com/guide/topics/manifest/meta-data-element
+      "metadata": {
+        "type": "array",
+        "items": { "$ref": "#/definitions/metadata" }
+      },
+      "stringResources": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": { "type": "string" },
+            "value": { "type": "string" }
+          }
+        }
+      },
+      "activities": {
+        "type": "array",
+        "items": { "$ref": "#/definitions/activity" }
+      }
+    },
+    "activity": {
+      "type": "object",
+      "properties": {
+        "allowEmbedded": { "type": "boolean" },
+        "allowTaskReparenting": { "type": "boolean" },
+        "alwaysRetainTaskState": { "type": "boolean" },
+        "autoRemoveFromRecents": { "type": "boolean" },
+        "banner": { "type": "string" },
+        "clearTaskOnLaunch": { "type": "boolean" },
+        "colorMode": { "enum": ["hrd", "wideColorGamut"] },
+        "configChanges": {
+          "enum": ["mcc", "mnc", "locale", "touchscreen", "keyboard", "keyboardHidden", "navigation", "screenLayout", "fontScale", "uiMode", "orientation", "density", "screenSize", "smallestScreenSize"]
+        },
+        "directBootAware": { "type": "boolean" },
+        "documentLaunchMode": { "enum": ["intoExisting", "always", "none", "never"]},
+        "enabled": { "type": "boolean" },
+        "excludeFromRecent": { "type": "boolean" },
+        "exported": { "type": "boolean" },
+        "finishOnTaskLaunch": { "type": "boolean" },
+        "hardwareAccelerated": { "type": "boolean" },
```

Comment:
This seems confusing - wouldn't the engine's rendering code override this anyway?

Reply:
These Activity definitions are for other Activitys, not the main Flutter Activity.  The doc with all the examples of stuff we need to support made a case for why you'd need to declare other Activitys for certain plugins.

Reply:
That makes sense - maybe it's worth clarifying in the description for activities that these are not to configure the main Flutter activity?